pycsw/core/util.py

```diff
@@ -33,11 +33,15 @@
 import time
 import datetime
 import logging
-import urllib2
+try:
+    from urllib2 import Request, urlopen
+except:
+    from urllib.request import Request, urlopen
 from shapely.wkt import loads
 from owslib.util import http_post
 from pycsw.core.etree import etree
 
+from six import text_type
```

Comment:
Forgive my Python 3 ignorance here, but if pycsw doesn't import six, and OWSLib does, can pycsw still use it?  If pycsw doesn't use six, what are the implications?

If things are too painful without using six, and six has packaging distribution support (which @kalxas agrees to), then I'm not against using it per se.  Is using six a temporary solution?

Again, apologies for my Python 3 ignorance, and huge thanks to @ocefpaf and @QuLogic to help bring Python 3 into pycsw.